src/app-layout/drawer/styles.scss

```diff
@@ -48,13 +49,11 @@ $drawer-z-index-mobile: 1001;
   }
   .drawer-closed > & {
     width: constants.$sidebar-size-closed;
-    &:hover {
-      background: awsui.$color-background-layout-panel-hover;
-    }
-    &.non-interactive {
+    &.drawer-content-clickable {
+      cursor: pointer;
+      color: awsui.$color-text-interactive-default;
```

Comment:
Made some refactoring in this file to more clearly connect styles and their reasons.

Why is there a pointer cursor? Not because the drawer is closed, but because it is clickable

Reply:
There could be closed non-clickable drawers and the new CSS structure allows that